README.md

```diff
@@ -1,6 +1,6 @@
 # boot-sassc
 
-[![Clojars Project](http://clojars.org/mathias/boot-sassc/latest-version.svg)](http://clojars.org/mathias/boot-sassc)
+[![Clojars Project](http://clojars.org/boot-sassc/latest-version.svg)](http://clojars.org/mathias/boot-sassc)
```

Comment:
You're right, I had misunderstood.

We want to link to https://clojars.org/boot-sassc

I fixed that now. It seems better

What do you think ?